mysqld_exporter.go

```diff
@@ -164,24 +238,76 @@ func (e *Exporter) scrape(scrapes chan<- []string) {
 	e.duration.Set(float64(time.Now().UnixNano()-now) / 1000000000)
 }
 
+func (e *Exporter) setGenericMetric(name string, value float64) {
+	if _, ok := e.metrics[name]; !ok {
+		e.metrics[name] = prometheus.NewUntyped(prometheus.UntypedOpts{
+			Namespace: namespace,
+			Name:      name,
+		})
+	}
+
+	e.metrics[name].Set(value)
+}
+
 func (e *Exporter) setMetrics(scrapes <-chan []string) {
-	for row := range scrapes {
+	var comRegex = regexp.MustCompile(`^com_(.*)$`)
+	var connectionErrorRegex = regexp.MustCompile(`^connection_errors_(.*)$`)
+	var handlerRegex = regexp.MustCompile(`^handler_(.*)$`)
```

Comment:
I don't think these make sense as one metric, they're all for very different things